components/Modal/CreateImageUpload.js

```diff
@@ -442,88 +555,515 @@ class CreateImageUploadModal extends React.Component {
-      name: `Upload to ${displayName}`,
-      steps: [uploadAuth(provider), uploadSettings(provider)]
-    });
+    const azureUploadSettings = {
+      name: "Destination",
+      component: (
+        <React.Fragment>
+          <Text className="help-block cc-c-form__required-text">
+            <FormattedMessage defaultMessage="All fields are required." />
+          </Text>
+          <Form isHorizontal className="cc-m-wide-label">
+            <div className="pf-c-form__group">
+              <div className="pf-c-form__label pf-m-no-padding-top pf-l-flex pf-u-display-flex pf-m-justify-content-flex-start pf-m-nowrap">
+                <label htmlFor="image-name-input" className="pf-l-flex__item">
+                  <span className="pf-c-form__label-text">
+                    <FormattedMessage defaultMessage="Image name" />
+                  </span>
+                  <span className="pf-c-form__label-required" aria-hidden="true">
+                    &#42;
+                  </span>
+                </label>
+                <Popover
+                  id="popover-help"
+                  bodyContent={
+                    <React.Fragment>
+                      <FormattedMessage defaultMessage="Provide a file name to be used for the image file that will be uploaded." />
+                    </React.Fragment>
+                  }
+                  aria-label="image name help"
+                >
+                  <Button variant="plain" aria-label="image name help">
+                    <OutlinedQuestionCircleIcon id="popover-icon" />
+                  </Button>
+                </Popover>
+              </div>
+              <TextInput
+                className="pf-c-form-control"
+                value={imageName}
+                type="text"
+                id="image-name-input"
+                onChange={this.setImageName}
+              />
+            </div>
+            <div className="pf-c-form__group">
+              <div className="pf-c-form__label pf-m-no-padding-top pf-l-flex pf-u-display-flex pf-m-justify-content-flex-start pf-m-nowrap">
+                <label htmlFor="container-input" className="pf-l-flex__item">
+                  <span className="pf-c-form__label-text">Storage container</span>
+                  <span className="pf-c-form__label-required" aria-hidden="true">
+                    &#42;
+                  </span>
+                </label>
+                <Popover
+                  id="bucket-popover"
+                  bodyContent={
+                    <TextContent>
+                      <FormattedMessage defaultMessage="Provide the Blob container to which the image file will be uploaded. You can find containers on the" />
+                      <strong>
+                        <FormattedMessage defaultMessage=" Blobs " />
+                      </strong>
+                      <FormattedMessage defaultMessage="page of a storage account. You can find storage accounts on the" />
```

Comment:
Would this statement make sense? I'm not seeing a specific "Blob" page. Open to suggestions on this one.
> You can find containers under the **Blob service** section of a storage account.

Reply:
Yes this makes sense.